mappings/net/minecraft/world/gen/feature/DefaultBiomeFeatures.mapping

```diff
@@ -139,9 +139,24 @@ CLASS net/minecraft/class_3864 net/minecraft/world/gen/feature/DefaultBiomeFeatu
 	FIELD field_22064 SOUL_SOIL Lnet/minecraft/class_2680;
 	FIELD field_22065 CRIMSON_ROOTS Lnet/minecraft/class_2680;
 	FIELD field_23076 NETHER_GOLD_ORE Lnet/minecraft/class_2680;
+	FIELD field_23851 MIXED_NETHER_SPRING_CONFIG Lnet/minecraft/class_4642;
+	FIELD field_23852 SOUL_SAND_SPRING_CONFIG Lnet/minecraft/class_4642;
+	FIELD field_23853 ENCLOSED_SOUL_SAND_SPRING_CONFIG Lnet/minecraft/class_4642;
+	FIELD field_23854 BASALT_COLUMN_CONFIG Lnet/minecraft/class_5156;
+	FIELD field_23855 TALL_BASALT_COLUMN_CONFIG Lnet/minecraft/class_5156;
+	FIELD field_23856 BASALT_BLOB_CONFIG Lnet/minecraft/class_5160;
+	FIELD field_23857 BLACKSTONE_BLOB_CONFIG Lnet/minecraft/class_5160;
+	FIELD field_23858 DELTA_CONFIG Lnet/minecraft/class_5158;
 	FIELD field_24110 VERY_RARE_BEEHIVES_CONFIG Lnet/minecraft/class_4659;
 	FIELD field_24111 REGULAR_BEEHIVES_CONFIG Lnet/minecraft/class_4659;
 	FIELD field_24112 MORE_BEEHIVES_CONFIG Lnet/minecraft/class_4659;
+	FIELD field_24682 JUNGLE_CONFIGURED_RUINED_PORTAL Lnet/minecraft/class_5312;
+	FIELD field_24683 SWAMP_CONFIGURED_RUINED_PORTAL Lnet/minecraft/class_5312;
+	FIELD field_24684 MOUNTAIN_CONFIGURED_RUINED_PORTAL Lnet/minecraft/class_5312;
+	FIELD field_24685 OCEAN_CONFIGURED_RUINED_PORTAL Lnet/minecraft/class_5312;
+	FIELD field_24686 NETHER_CONFIGURED_RUINED_PORTAL Lnet/minecraft/class_5312;
+	FIELD field_24711 STANDARD_CONFIGURED_RUINED_PORTAL Lnet/minecraft/class_5312;
+	FIELD field_24712 DESERT_CONFIGURED_RUINED_PORTAL Lnet/minecraft/class_5312;
```

Comment:
Why did you name these with `CONFIGURED`?

Reply:
There isn't any precedent with configured features in this class, but since the feature configs have the `_CONFIG` suffix, I added it to the ruined portal configured features to be consistent in detail.

Reply:
I guess we can go with @kroppeb and drop `CONFIGURED` from name, as there is no version that is not configured.